WORKSPACE

```diff
@@ -35,10 +36,10 @@ bind(
 # 1. Determine SHA256 `wget https://github.com/envoyproxy/envoy-wasm/archive/$COMMIT.tar.gz && sha256sum $COMMIT.tar.gz`
 # 2. Update .bazelrc and .bazelversion files.
 #
-# envoy-wasm commit date: 10/09/2019
-ENVOY_SHA = "ec03328688895d99c9f5ae4fd7f8459ef3e95212"
+# envoy-wasm commit date: 10/11/2019
+ENVOY_SHA = "29b71643c999af4e31ba86d41740edbef71c73fe"
```

Comment:
can you use the one from Greg's PR? or is this more recent?

Reply:
it's more recent